tests/src/test/java/alluxio/stress/StressClientIOBenchIntegrationTest.java

```diff
@@ -0,0 +1,137 @@
+  public TemporaryFolder mFolder = new TemporaryFolder();
+
+  @Test
+  public void readArray() throws Exception {
+    // Only in-process will work for unit testing.
+    String output1 = new StressClientIOBench().run(new String[] {
+        "--in-process",
+        "--tag", "ReadArray-NOT_RANDOM",
+        "--start-ms", Long.toString(System.currentTimeMillis() + 1000),
+        "--base", sLocalAlluxioClusterResource.get().getMasterURI() + "/",
+        "--operation", "ReadArray",
+        "--threads", "2",
+        "--file-size", "1m",
+        "--block-size", "128k",
+        "--warmup", "0s", "--duration", "1s",
+    });
+
+    String output2 = new StressClientIOBench().run(new String[] {
+        "--in-process",
+        "--read-random",
+        "--tag", "ReadArray-RANDOM",
+        "--start-ms", Long.toString(System.currentTimeMillis() + 1000),
+        "--base", sLocalAlluxioClusterResource.get().getMasterURI() + "/",
+        "--operation", "ReadArray",
+        "--threads", "2",
+        "--file-size", "1m",
+        "--block-size", "128k",
+        "--warmup", "0s", "--duration", "1s",
+    });
+    generateAndVerifyReport(Arrays.asList("ReadArray-NOT_RANDOM", "ReadArray-RANDOM"), output1,
+        output2);
+  }
+
+  @Test
+  public void posRead() throws Exception {
+    // Only in-process will work for unit testing.
+    String output1 = new StressClientIOBench().run(new String[] {
+        "--in-process",
+        "--tag", "PosRead-NOT_RANDOM",
+        "--start-ms", Long.toString(System.currentTimeMillis() + 1000),
+        "--base", sLocalAlluxioClusterResource.get().getMasterURI() + "/",
+        "--operation", "PosRead",
+        "--threads", "2",
+        "--file-size", "1m",
+        "--block-size", "128k",
+        "--warmup", "0s", "--duration", "1s",
+    });
+
+    String output2 = new StressClientIOBench().run(new String[] {
+        "--in-process",
+        "--read-random",
+        "--tag", "PosRead-RANDOM",
+        "--start-ms", Long.toString(System.currentTimeMillis() + 1000),
+        "--base", sLocalAlluxioClusterResource.get().getMasterURI() + "/",
+        "--operation", "PosRead",
+        "--threads", "2",
+        "--file-size", "1m",
+        "--block-size", "128k",
+        "--warmup", "0s", "--duration", "1s",
+    });
+    generateAndVerifyReport(Arrays.asList("PosRead-NOT_RANDOM", "PosRead-RANDOM"), output1,
+        output2);
+  }
+
+  private void generateAndVerifyReport(List<String> expectedGraphNames, String... outputJson)
+      throws Exception {
+    List<String> args = new ArrayList<>();
+
+    // write out the jsons outputs to files (for the input of generate)
+    for (String output : outputJson) {
+      File input = mFolder.newFile();
+      try (FileWriter writer = new FileWriter(input)) {
+        writer.write(output);
+      }
+      args.add("--input");
+      args.add(input.getAbsolutePath());
+    }
+
+    // generate the output
+    File output = mFolder.newFile("report.html");
```

Comment:
I think we should be using different name then output since output is used multiple times here to mean different things.

Reply:
Renamed